arrows/ocv/inpaint.h

```diff
@@ -0,0 +1,55 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_ARROWS_OCV_INPAINT_
+#define KWIVER_ARROWS_OCV_INPAINT_
+
+#include <arrows/ocv/kwiver_algo_ocv_export.h>
+
+#include <vital/algo/image_filter.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ocv {
+
+/// OCV image inpainting process
+///
+/// Create a masked RGB image from a RGBA image
+class KWIVER_ALGO_OCV_EXPORT inpaint
+  : public vital::algo::image_filter
+{
+public:
+  PLUGIN_INFO( "ocv_inpainting",
+               "Inpaint a RGBA image using non-zero A values as a mask." )
+
+  inpaint();
+  virtual ~inpaint();
+
+  /// Get this algorithm's \link vital::config_block configuration block
+  /// \endlink
+  virtual vital::config_block_sptr get_configuration() const;
```

Comment:
I think these can be marked `override`?
```suggestion
  vital::config_block_sptr get_configuration() const override;
```